src/bin/cql-stress-scylla-bench/args.rs

```diff
@@ -100,6 +101,8 @@ where
         true,
         "use compression for client-coordinator communication",
     );
+    let shard_connection_count =
+        flag.u64_var("connection-count", 1, "number of connections per shard");
```

Comment:
Let's change the name of the parameter to indicate that this is a per-shard connection count.

Reply:
What about the `connection-count` parameter (which was just a placeholder until now)? The commands in `args_test.in` include such parameter.

We can:
- either replace `connection-count` with `shard-connection-count`
- or include both of them (still leaving `connection-count` as a placeholder since original s-b supports such parameter)

Reply:
The other idea I have is to have a parameter which defines the number of connections, as well as a type (per-shard/per-node). So something like `--connection-count per-shard(10)` or `--connection-count per-node(10)`.

Reply:
> What about the connection-count parameter (which was just a placeholder until now)? The commands in args_test.in include such parameter.

Let's introduce the new `shard-connection-count` and keep ignoring the old one.

> The commands in `args_test.in` include such parameter.

If we are going to replace s-b in scylla-cluster-tests, I'd rather not change the existing behavior of the tool, so I'd rather keep ignoring the existing option.